tests.py

```diff
@@ -132,9 +133,10 @@ def testDetectStorage(self):
   def testGetContainerInfo(self):
```

Comment:
personal preference suggestion: add a docstring so you can see this in the unittest output what is being tested

Reply:
Done